features/regexp.yml

```
@@ -0,0 +1,44 @@
+name: Regular Expressions (RegExp)
+description: The RegExp object is used for matching text with a regular expression.
```

Feature names are sentence case and also-known-as text goes in the description (if needed, though not in this case)

```suggestion
name: Regular expressions
description: The `RegExp` object represents a regular expression, a notation for matching text patterns.
```